cypress/integration/providers_spec.js

```diff
@@ -30,7 +30,7 @@ describe('Dashboard Providers Page', () => {
 
     it('should display a link to view providers', () => {
```

Comment:
Can we add a test in one of the specs where we navigate directly to a "bookmarked url" and then verify that the Datepicker on that page reflects the state of the input URL?

Reply:
Added